### DISCUSSION TITLE
Increasing script speed

### DISCUSSION THREAD

**Discussion by pedrokkrause** [Stellarium/stellarium#2344] [Q&A] [answered]
When running a script, it is necessary to use core.wait() when changing the date (otherwise things won't work properly), which limits a lot the speed of the script, specially when involving loops, since it seems that the minimum value for such function is around 0.01s. Is there a way to go faster, even if it require that the visible sky doesn't update? I need to generate some data regarding the sun's altitude for each day throughout the years.

**Comment by gzotti** [comment, marked as answer]
No, it is not possible, otherwise we would not say you need to add the wait().
